pod/perldelta.pod

```diff
@@ -77,6 +77,33 @@ XXX For a release on a stable branch, this section aspires to be:
 
 XXX Any deprecated features, syntax, modules etc. should be listed here.
 
+=head2 Downgrading a C<use VERSION> statement to below v5.11
+
+Attempting to issue a second C<use VERSION> statement that requests a version
+below than C<v5.11> when an earlier statement that requested a version higher
```

Comment:
`requests a version below than...` doesn't quite make sense. Maybe `requests a version lower than` (later we say `higher than`

Reply:
Ahyes.